### DISCUSSION TITLE
Lost 2FA Credentials and Recovery Codes

### DISCUSSION THREAD

**Discussion by abdullahkhfbII** [community/community#144758] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello, I’m experiencing an issue with accessing my GitHub account, and I hope someone here can help me or guide me through the recovery process.

**Issue:**
- I’ve enabled two-factor authentication (2FA) on my account, but I lost access to both my 2FA device and my recovery codes when I installed a **new OS.**
- As a result, I am cannot log into my account.

**What I’ve tried so far:**
- I’ve checked my email for any backup recovery options but didn’t find anything helpful.
- I’ve already reached out to GitHub support, but I’m still waiting for a response.
![image](https://github.com/user-attachments/assets/b20006dd-4a03-4102-9f37-69151b63d30c)

Thank you in advance for any advice or assistance you can provide.

**Comment by davevad93** [comment, marked as answer]
Hi @abdullahkhfbII ,

Considering that this is an account related problem, you have to wait for a response to the ticket you have opened to GitHub Support because that's the only way to get a resolution for this kind of issues, they can take a while to reply, even weeks, so be patient. I hope you'll get a resolution as soon as possible.

**Reply by abdullahkhfbII** [reply]
Thank you

**Comment by ebndev** [comment]
Hi @abdullahkhfbII,

As @davevad93 mentioned, you have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
